mlflow/crewai/autolog.py

```diff
@@ -19,14 +20,20 @@ def patched_class_call(original, self, *args, **kwargs):
 
     if config.log_traces:
         fullname = f"{self.__class__.__name__}.{original.__name__}"
+        span_type = _get_span_type(self)
         with mlflow.start_span(
```

Comment:
```suggestion
        with mlflow.start_span(name=fullname, span_type=span_type) as span:
```

nit: You may perhaps have different linter in the IDE? The repo's [rule](https://github.com/mlflow/mlflow/blob/master/pyproject.toml#L154) allow max 100 chars in a line.

Reply:
I don't remember if I changed here due to linter or if this is my style preference.. let me check